examples/template/README.md

```diff
@@ -0,0 +1,47 @@
+# [Dataset Name]
+
+<!-- Replace the title with the dataset name. -->
+
+## Dataset Description
+
+<!-- A concise description of the dataset. -->
+
+#### Citation
+
+```
```

Comment:
Add the bullet points `Original Source`, `Current Version`, and `Previous Versions`, where the latter two are optional. See [cora](https://github.com/Graph-Learning-Benchmarks/gli/tree/main/datasets/cora) as an example.

Reply:
Could you please give a definition of these versions?

Reply:
Original source refers to the first work that created the dataset. Current version refers to the work that is directly responsible for the exact form of the dataset stored in this folder. Previous versions refer to any intermediate versions between original source and the current version. There could be multiple citations for the previous versions.